tests/sentry/integrations/bitbucket_server/test_repository.py

```diff
@@ -66,16 +66,27 @@ def test_compare_commits(self):
             json=COMPARE_COMMITS_EXAMPLE,
         )
 
+        responses.add(
+            responses.GET,
+            "https://bitbucket.example.com/rest/api/1.0/projects/sentryuser/repos/newsdiffs/commits/e18e4e72de0d824edfbe0d73efe34cbd0d01d301/changes",
+            json=COMMIT_CHANGELIST_EXAMPLE,
```

Comment:
@pierredup I think we should have more unit tests than just the case for `MODIFY`. Ideal world, we'd have unit tests for all 4 cases. But we should definitely have a test case for `MOVE` since it's a bit different than the rest.